docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,165 @@
+   * It gets the workload set that has the specified version.
+   * From the workload set, it gets the manifest version of the specified workload.
+   * It installs the manifest version of the workload.
+   * It stays in workload-set update mode when it's done.
+
+1. ```dotnetcli
+   dotnet workload install maui-ios
+   ```
+
+   This command installs the workload set version `9.0.100-preview.7.24414.1`, since the preceding `install` command example pinned that workload set.
+
+Using `--version` with either `install` or `update` pins `install` to the specified version, but `update` is only pinned to workload-set update mode, not to a workload-set version. If you then run `dotnet workload update` without the `--version` option, the `update` command:
+
+* Updates workloads to the latest available workload-set version.
+* "Unpins" the `install` command.
+* Stays in workload-set update mode.
+
+## Update using latest workload set
+
+To update installed workloads to the latest workload-set version available on the configured feeds, run the following commands:
+
+1. ```dotnetcli
+   dotnet workload config --update-mode workload-set
+   ```
+
+   The preceding command is necessary only if you are currently in manifests update mode. If you don't know, [check the current update mode](#check-the-update-mode-and-version). Workload-set update mode is the default starting in .NET 9 SDK.
+
+1. ```dotnetcli
+   dotnet workload update
+   ```
+
+   In workload set update mode, this command updates workloads to the latest workload set version, unless you have specified the workload set version in global.json.
+
+## Update to a workload-set version
+
+To specify a workload set version to update to when you're not specifying it in global.json, use the `--version` option of the `update` command. For example:
+
+```dotnetcli
+dotnet workload update --version 8.0.400
+```
+
+Workload-set update mode will be selected if it wasn't already selected.
+
+## Use global.json for the workload set version
+
+Create a `global.json` file to specify the workload set version for a repository. The format looks like this example:
+
+```json
+{
+  "sdk": {
+    "workloadVersion": "9.0.100-preview.7.24414.1"
+  }
+}
+```
+
+With the current directory in the same repository and the CLI in workload set update mode, the `install`, `update`, and `restore` commands install workloads for the specified workload set version. If you don't have a global.json file, and you're in workload set update mode, the `restore` command installs the workload set version that was established when you switched from manifests update mode to workload sets update mode.
+<!--todo:what about if it's SDK 9 so you defaulted to workload set update mode, and you never established a workload set version -- does it go for latest available then?-->
+<!--todo:if you're in manifests update mode, you stay in that mode and it doesn't seem to do a workload set update, i.e., it seems to ignore global.json(?)-->
```

Comment:
if you have a workload set in the global.json, it will treat you as in workload set mode even if you have not run the config command or --version. global.json overrides those.